distrobox-create

```diff
@@ -505,6 +507,13 @@ generate_command() {
 	# Add additional flags
 	result_command="${result_command} ${container_manager_additional_flags}"
 
+	# Add init-time environment variables. The only requirement is that
+	# their name begins with DBX_CONTAINER_INIT_. They'll (likely!) be
+	# consumed by container-init, or ingeneral inside of the distrobox.
+	for i in $(printenv | grep '=' | grep -Ev ' |"' | grep '^DBX_CONTAINER_INIT_'); do
+		result_command="${result_command} --env \"$i\""
```

Comment:
We will need curly braces around the variable or it won't pass the linting

Reply:
Yep, forgot to re-run shellcheck locally before pushing! Sorry :-)